.github/scripts/veristat-compare.py

```diff
@@ -0,0 +1,150 @@
+import sys
+import csv
+import os
+
+# File format:
+#
+# Columns:
+#  0. file_name
+#  1. prog_name
+#  2. verdict_base
+#  3. verdict_comp
+#  4. verdict_diff
+#  5. total_states_base
+#  6. total_states_comp
+#  7. total_states_diff
+#
+# Records sample:
+#  file-a,a,success,failure,MISMATCH,12,12,+0 (+0.00%)
+#  file-b,b,success,success,MATCH,67,67,+0 (+0.00%)
+
+TRESHOLD_PCT = 0
+
+HEADERS = ['file_name', 'prog_name', 'verdict_base', 'verdict_comp',
+           'verdict_diff', 'total_states_base', 'total_states_comp',
+           'total_states_diff']
+
+FILE        = 0
+PROG        = 1
+VERDICT_OLD = 2
+VERDICT_NEW = 3
+STATES_OLD  = 5
+STATES_NEW  = 6
```

Comment:
This could be wrapped in an [enum](https://docs.python.org/3/library/enum.html)

Reply:
These are used as indices in a vector, using `Enum` would make the notation less readable: `row[FILE]` &rarr; `row[FILE.Value]`, so I'd rather not use enum in this case.